job-dsls/jobs/prod_offline_repo_builder.groovy

```diff
@@ -0,0 +1,46 @@
+/**
+ * Check offliner manifest against offline repository
+ */
+import org.kie.jenkins.jobdsl.Constants
+
+String commands = this.getClass().getResource("job-scripts/prod_offline_repo_builder.sh").text
+
+job("offline-repo-builder") {
+    description("Check offliner manifest against offline repository")
+
+    logRotator {
+        numToKeep(5)
+        artifactNumToKeep(1)
+    }
+
+    parameters {
+        stringParam("MANIFEST_URL", "", "Offliner manifest URL")
+        stringParam("RELEASE_REPO_URL", "http://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8081/nexus/content/repositories/offline-repo-7.6/", "Scratch repository for the release")
```

Comment:
is that URL of repo correct @winklerm ? i am confused by the number 7.6 only ;)

Reply:
Good point - it is outdated; on the other hand, this value is going to be changed for each release, so this particular URL is just a default that needs to be overwritten when running the job.

Anyway, I guess we could put a recent value there, or - maybe better - something like:

```suggestion
        stringParam("RELEASE_REPO_URL", "http://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8081/nexus/content/repositories/offline-repo-REPLACE-ME-WITH-MAJOR.MINOR/", "Scratch repository for the release")
```

Wdyt?

Reply:
I went with the update of the parameters to recent values as many times they serve as both a default value and an example. I think it's simpler to understand than using place holders or anything similar.